### PR TITLE
Update Storage diagram, DG for Add and Delete Schedule

### PR DESCRIPTION
**Pull request by dhafinrazaq**

### CODE REVIEW

**Comment by dhafinrazaq** [comment]
Can anyone check if this is correct? The new diagram is a bit different
Before:
Screen Shot 2020-10-19 at 1 10 52 PM

Now:
Screen Shot 2020-10-19 at 1 09 45 PM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -516,45 +555,65 @@ testers are expected to do more *exploratory* testing.
...
+1. Adding a Schedule while all Clients and Sessions are being shown
+
+   1. Prerequisites: Multiple Clients and Sessions in the list can be viewed on the left and right panel of the GUI respectively.
+   1. Test case: `schadd c/1 s/1`<br>
+      Expected: Add a Schedule associated with Client of index 1 in the Client List and Session of index 1 in the Session List.
+      Details of the added Schedule is shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `schadd s/1 c/2`<br>
+      Expected: Add a Schedule associated with Client of index 2 in the Client List and Session of index 1 in the Session List.
+      Details of the added Schedule is shown in the status message. Timestamp in the status bar is updated.
+   1. Other incorrect edit commands to try: `schadd c/1`, `schadd c/0 s/2`, `schadd c/x s/y`, `...` (where x is larger than the Client List size or y is larger than the Session List size)<br>
```
Comment (benclmnt) [inline docs/DeveloperGuide.md]: typo - add commands

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -516,45 +555,65 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No Schedule is added. Error details shown in the status message.
+
 ### Editing a Schedule
 
 1. Editing a Schedule while all Schedules are being shown
 
    1. Prerequisites: Multiple Schedules in the list can be viewed on the main panel of the GUI.
-
-   1. Test case: `editschedule c/1 s/1 us/2`<br>
+1. Test case: `editschedule c/1 s/1 us/2`<br>
```
Comment (benclmnt) [inline docs/DeveloperGuide.md]: it's changed from `editschedule`  -> `schedit`

**Review by benclmnt — COMMENTED** [review]
Looks good generally

**Comment by dhafinrazaq** [comment]
> Looks good generally

resolved

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -428,6 +464,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+   Use case ends.
+
+**Extensions**
+
+- 2a. The Schedule to be added is overlapping with another Schedule
+
+  - 2a1. FitEgo shows an error message
+
+    Use case resumes at step 2.
```
Comment (tanweijie123) [inline docs/DeveloperGuide.md]: Want to update to this? 
image

**Review by tanweijie123 — APPROVED** [review]
👍

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -547,45 +624,65 @@ testers are expected to do more *exploratory* testing.
...
       Expected: Similar to previous.
 
+### Adding a Schedule
+
+1. Adding a Schedule while all Clients and Sessions are being shown
+
+   1. Prerequisites: Multiple Clients and Sessions in the list can be viewed on the left and right panel of the GUI respectively.
+   1. Test case: `schadd c/1 s/1`<br>
+      Expected: Add a Schedule associated with Client of index 1 in the Client List and Session of index 1 in the Session List.
+      Details of the added Schedule is shown in the status message. Timestamp in the status bar is updated.
```
Comment (kelvinvin) [inline docs/DeveloperGuide.md]: Can remove the timestamp sentence

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -547,45 +624,65 @@ testers are expected to do more *exploratory* testing.
...
 ### Editing a Schedule
 
 1. Editing a Schedule while all Schedules are being shown
 
    1. Prerequisites: Multiple Schedules in the list can be viewed on the main panel of the GUI.
-
-   1. Test case: `editschedule c/1 s/1 us/2`<br>
+1. Test case: `schedit c/1 s/1 us/2`<br>
       Expected: Edit Schedule with client index 1 and session index 1 is edited to session index 2.
       Details of the edited schedule is shown in the status message. Timestamp in the status bar is updated.
```
Comment (kelvinvin) [inline docs/DeveloperGuide.md:724]: Can remove the timestamp sentences in this section too

Comment (dhafinrazaq) [reply docs/DeveloperGuide.md:724]: i think maguire is editing this part

**Pull request merged**